host/xtest/xtest_main.c

```diff
@@ -41,6 +41,7 @@ ADBG_SUITE_DEFINE(ffa_spmc);
 #endif
 ADBG_SUITE_DEFINE(regression);
 
+char *xtest_argv_0;
```

Comment:
As I said in the other PR, I'd prefer `xtest_progname` as that's the expected use.